package org.example.day13.except;

public class ExceptMain {
    public static void main(String[] args) {
        ErrorClass error = new ErrorClass();
        error.call();
        System.out.println("에러발생후 실행될까요..!!!!!");
        System.out.println("에러발생후 실행될까요..!!!!!");
        System.out.println("에러발생후 실행될까요..!!!!!");
        System.out.println("에러발생후 실행될까요..!!!!!");
    }
}
